pkgs/pre-commit/pre-commit/registry.yaml

```diff
@@ -0,0 +1,9 @@
+packages:
+  - type: github_release
+    repo_owner: pre-commit
+    repo_name: pre-commit
+    description: A framework for managing and maintaining multi-language pre-commit hooks
+    version_constraint: "false"
+    version_overrides:
+      - version_constraint: "true"
+        no_asset: true
```

Comment:
Same with https://github.com/aquaproj/aqua-registry/pull/26126/files#r1725946370 .
We need to fix this somehow.

Reply:
https://github.com/aquaproj/aqua-registry/commit/942386740ac943a7ec911bec9baa14a3af0e8d3a Fixed.